src/modules/routes/constants/components.js

```diff
@@ -79,3 +79,9 @@ export const Reporting = asyncComponent({
   loader: () => import(/* webpackChunkName: 'reporting' */ 'modules/reporting/components/reporting-view/reporting-view')
     .then(module => module.default)
 })
+
+export const Report = asyncComponent({
```

Comment:
If I remember correctly, this file is meant for just top-level views. If that's correct, you shouldn't be adding this here (the code directly above, for `moduleName: 'Reporting',` should be sufficient). @stephensprinkle Is that correct?

Reply:
That’s correct + been the convention thus far.  What these do is establish Webpack breakpoints, so this is where you get discrete files in the build.  

Let’s keep to this convention for now, but later we can potentially establish additional breakpoints.  For now, just be declarative when it comes to routes within their respective components.